backend/novion.py

```
@@ -125,16 +159,111 @@ def process_query(query: str):
+
+# New streaming function
+async def stream_query(query: str) -> AsyncGenerator[str, None]:
+    """
+    Stream user query responses as they become available.
+    
+    This is a generator function that yields chunks of the response
+    as they become available from different agents.
+    """
+    print(f"Starting stream_query with query: {query}")
+    
+    results = []
```

Ruff (0.8.2)

Local variable `results` is assigned to but never used

Remove assignment to unused variable `results`

(F841)